### ISSUE TITLE
Things that should be in the options

### ISSUE THREAD

**Issue by 0xpr03** [0xpr03/VocableTrainer-Android#31]
Tracking issue for things that should be configurable. Please leave a comment for things that should be added to an options section of the app.

- [ ] Addition auto-next time

**Comment by OzWT** [comment]
Maybe this is not the right place here, or it is already configurable and I'm to stupid.
Here is my wish for extending the configuration of the import format:
In the custom format import, it is possible to use a single sign as the delimiter, but it is not possible to use a tab as the delimiter. I tried "\t" but I could only set one character.

**Comment by 0xpr03** [comment]
> I tried "\t" but I could only set one character.

Yes, `\t` is the standard in sourcecode, but this means the compiler is replacing it with the correct character. I would have to interpret your input, which I'm currently not doing. I'm not sure how to do this in a user friendly way, as we still have to enforce the one character limit.  
What you can do, is to copy a tab character in your smartphone and paste it in the dialog.

> a tab as the delimiter

Currently you can go with the [TDF Preset](https://github.com/0xpr03/VocableTrainer-Android/wiki/CSV-Format#tdf). I hope it's fitting your format requirements.

**Comment by OzWT** [comment]
The TDF Preset was the first thing I have tried. But the multi-translation delimiter doesn't match. In my case I need a "," as the multi-translation delimiter. So I tried the custom format.
The copy / paste hint for the tab character was very good. Thanks! So I don't need to modify my CSV-lists.

Maybe you can use a drop down menu for delimiters. The custom format settings are much more than only set the delimiter. In most cases, only a few separators are required. And for users who need a special delimiter, one point in the drop down menu could be "other character", which opens the 1 character restricted input window.

**Comment by 0xpr03** [comment]
> drop down menu

That's an interesting approach I'll have to play around with.
